apis/policies/v1/admissionpolicy_webhook.go

```diff
@@ -57,7 +56,7 @@ func (r *AdmissionPolicy) Default() {
 	}
 }
 
-//+kubebuilder:webhook:path=/validate-policies-kubewarden-io-v1alpha2-admissionpolicy,mutating=false,failurePolicy=fail,sideEffects=None,groups=policies.kubewarden.io,resources=admissionpolicies,verbs=create;update,versions=v1alpha2,name=vadmissionpolicy.kb.io,admissionReviewVersions={v1,v1beta1}
+//+kubebuilder:webhook:path=/validate-policies-kubewarden-io-v1-admissionpolicy,mutating=false,failurePolicy=fail,sideEffects=None,groups=policies.kubewarden.io,resources=admissionpolicies,verbs=create;update,versions=v1alpha2;v1,name=vadmissionpolicy.kb.io,admissionReviewVersions={v1,v1beta1}
```

Comment:
This line still has a `v1alpah2` occurrence inside of it. Is that a leftover or is it intentional?

Reply:
I left it intentionally because I think Webhook should watch both versions (old versions stored in etcd are still v1alpha2, even though it will be transformed to v1 when you fetch them)
To be honest it might work fine with just v1, but I haven’t tested it

Reply:
removed [here](https://github.com/kubewarden/kubewarden-controller/pull/247/commits/84a0ba1a5d5a925b1cafd7e40e4ce9892dc1a62e) I did some manual test and it is working fine

Reply:
I removed them and it seems to be fine. It looks like it will fetch and transform v1alpha2 to v1 if we add the webhook just for v1